subt/ros/robot/launch/freyja.launch

```diff
@@ -174,7 +174,7 @@
   <node pkg="topic_tools" type="relay" name="points_merge_front" args="$(arg robot_name)/rgbd_front/points $(arg robot_name)/rgbd_all/points"/>
   <node pkg="topic_tools" type="relay" name="points_merge_rear" args="$(arg robot_name)/rgbd_rear/points $(arg robot_name)/rgbd_all/points"/>
 
-  <node ns="mapping" pkg="octomap_server" type="octomap_server_node" name="octomap">
+  <anode ns="mapping" pkg="octomap_server" type="octomap_server_node" name="octomap">
```

Comment:
`anode` is for me too close to `node` ... I know that you plan to comment this out - do there is no need "structure file" in ROS (validation schema?)?
I would comment it out with <!-- (or how it is) ... or use some larger `dummy_node` or `disabled_node` instead of `anode`?

Reply:
As described in the related commit message of 3ea1da1a1a61f8ceba0aad9fcb7013a5ebad7e7c, commenting out is not possible since it seems that comment cannot be recursive (comments inside comments). I am testing a variant with `if="0"` attribute which seems to be the ROS way to do this. However the build of the base image takes forever so it is going to take some time.

Reply:
http://wiki.ros.org/roslaunch/XML#if_and_unless_attributes

Reply:
It works dae77d0